backend/src/main.ts

```diff
@@ -10,6 +10,15 @@ async function bootstrap() {
 
   const app = await NestFactory.create(AppModule, { cors: true });
 
+  const options = new DocumentBuilder()
+  .setTitle('Gesundheitsportal-Backend')
+  .setDescription('Dokumentation für das Gesundheitsportal-Backend')
+  .setVersion('1.0')
+  .build();
+
+  const document = SwaggerModule.createDocument(app, options);
+  SwaggerModule.setup('/docs', app, document);
+
```

Comment:
Die Dokumentation hat im Master nix verloren

Reply:
Die Dokumentation ist noch immer vorhanden.
Mach bitte wenigstens eine abfrage um den Code, der prüft ob du in deiner Dev-Umgebung bist...

Reply:
Es wurde jetzt komplett entfernt.
Ich habe das SwaggerModule nochmal benötigt für ein paar Bugfixes und nach der Nutzung vergessen zu entfernen.